### PR TITLE
feat: add logger and extractor docs

### PR DESCRIPTION
**Pull request by StewartJingga**

### CODE REVIEW

File: cmd/rundir.go [hunk 1]
```diff
@@ -13,19 +12,19 @@ func rundir(dirPath string) {
...
 
 	reader := recipe.NewReader()
 	recipeList, err := reader.ReadDir(dirPath)
 	if err != nil {
 		panic(err)
 	}
-	runner, cleanFn := initRunner(c)
+	runner, cleanFn := initRunner(c, log)
 	defer cleanFn()
 	faileds, err := runner.RunMultiple(recipeList)
```
Comment (GrayFlash) [inline cmd/rundir.go:27]: ```faileds``` can be renamed to ```failed_recipes``` or ```failures```, wdyt??

Comment (StewartJingga) [reply cmd/rundir.go:27]: yup definitely can be changed.
how about to `failedRecipes` for following convention?

Comment (GrayFlash) [reply cmd/rundir.go:27]: Yup seems better

**Comment by GrayFlash** [comment]
@StewartJingga seems fine to me!!

**Pull request merged**